### ISSUE TITLE
Messages 

### ISSUE THREAD

**Issue by owencz1998** [TopiSenpai/Reddit-Discord-Bot#6]
Can there be a option between having embedded and normal message to allow videos to work.

**Comment by TopiSenpai** [comment]
I guess I can add that do you have any specific message layout in mind?

You can build one with something like this

https://autocode.com/tools/discord/embed-builder/

**Comment by owencz1998** [comment]
Yeah just keep the information that would be cool also did u create the bot with autocode as I have premium haha

**Comment by TopiSenpai** [comment]
The bot is not built in any way with autocode

This was just so you can build an example message layout

**Comment by owencz1998** [comment]
Ahh ok its literally what u have now but not as a embedded message so video links work to

**Comment by TopiSenpai** [comment]
The embed consists of 4 parts
The title
The URL
The description
The fields

**Comment by owencz1998** [comment]
Could 

It have the 

Title 
Description 
 url

**Comment by topi314** [comment]
I plan to actually look into this tomorrow finally :)

**Comment by topi314** [comment]
![image](https://github.com/topi314/Reddit-Discord-Bot/assets/15636011/1c7d60da-8082-4b2c-b3df-1ce6cdd6bf51)
that's how it's looking now, not sure if I like it tbh lol

**Comment by topi314** [comment]
I think I found a decent layout
![image](https://github.com/topi314/Reddit-Discord-Bot/assets/15636011/b18d326f-fdd6-42be-8e2e-5468815e2461)

**Comment by topi314** [comment]
released in https://github.com/topi314/Reddit-Discord-Bot/releases/tag/v2.0.0-beta.2
you can update existing webhooks via:
```
/reddit update <subreddit> Text
```
